src/scene.js

```diff
@@ -911,6 +944,9 @@ Scene.prototype.preProcessSceneConfig = function () {
         }
     }
 
+    // TODO, find a home for this
+    this.loadDataSources();
```

Comment:
I think to function properly, this needs to go in `updateConfig`, which instantiates various objects after the scene config is parsed.